core/src/main/java/org/fao/geonet/api/records/attachments/CMISStore.java

```diff
@@ -128,7 +128,7 @@ protected MetadataResource createResourceDescription(final ServiceContext contex
         }
 
         MetadataResourceExternalManagementProperties metadataResourceExternalManagementProperties =
-            getMetadataResourceExternalManagementProperties(context, metadataId, metadataUuid, visibility, resourceId, filename, document.getVersionLabel(), document.getVersionSeriesId(), document.getType());
+            getMetadataResourceExternalManagementProperties(context, metadataId, metadataUuid, visibility, resourceId, filename, document.getVersionLabel(), document.getVersionSeriesId(), document.getType(), MetadataResourceExternalManagementProperties.ValidationStatus.UNKNOWN);
```

Comment:
I don't think the default should MetadataResourceExternalManagementProperties.ValidationStatus.UNKNOWN.
Shouldn't it use cmisConfiguration.getExternalResourceManagementStatusDefaultValue()?

I do see that cmisConfiguration.getExternalResourceManagementStatusDefaultValue() can potentially return null  so I think you would need another function that would be based on that value.

If the value is null then  I believe the default value should be based on the usage of the status field.
i.e. if the status field is used then the default should be MetadataResourceExternalManagementProperties.ValidationStatus.INCOMPLETE - because if the status field exists then it means that there is validation being done by the third party and I would expect that it would be validating other metadata that is not supplied during the basic file upload. -i.e. description would not be supplied hence it would be INCOMPLETE.
But if the status field is not being used then there is no validation being performed so we will never know the status so it will always be UNKNOWN

If the default value is not null then it should use that value for the default.

Reply:
There is no way to retrieve existing backend status. The current CmisClient in the CmisStore has some issue and this is the blocker for it